wp-seo-main.php

```diff
@@ -308,91 +309,10 @@ function wpseo_frontend_head_init() {
 }
 
 /**
- * Register the promotion class for our GlotPress instance
- *
- * @link https://github.com/Yoast/i18n-module
- *
- * @return yoast_i18n
- */
-function register_i18n_promo_class() {
```

Comment:
I see no reason why anyone would use this call directly outside of this plugin, as you can't change one thing about it... Which is why I'm just moving it.